### DISCUSSION TITLE
Pinia ORM axios

### DISCUSSION THREAD

**Discussion by starredev** [CodeDredd/pinia-orm#1900] [General]
What is the correct implementation for axios in pinia orm? I followed the documentation on the main website but i dont get it working. whatever i'm trying..

retrieving [Pinia ORM Axios] The axios instance is not registered. Please register the axios instance to the repository.

https://stackblitz.com/edit/vue3-vite-starter-7hutpf

**Comment by starredev** [comment]
I bought you a coffee for the solution @CodeDredd as long you need it :) <3

**Reply by CodeDredd** [reply]
Ohhh that's nice. Thank you! ❤️ .... looking into it for you right now....i think i have a mistake in the plugin definition.

**Reply by CodeDredd** [reply]
ok i had to change the plugin registration. Please look into the updated docs and checkout the edge channel to test it out.

**Reply by starredev** [reply]
> ok i had to change the plugin registration. Please look into the updated docs and checkout the edge channel to test it out.

I have tested it and i am getting the same error. Reproduced on https://stackblitz.com/edit/vue3-vite-starter-7hutpf
Could you take a look when you have time :)?

**Reply by CodeDredd** [reply]
You are right. Fixed it for now. You test it with the edge channel.

**Reply by starredev** [reply]
> You are right. Fixed it for now. You test it with the edge channel.

Almost! I cloned pinia-orm repo and used playground vue3vite. Now I'm getting the following error

Promise {<rejected>: TypeError: Cannot read properties of undefined (reading 'axios')
    at get axios (http://localhost…}
index.mjs:147 
 Uncaught (in promise) 
TypeError: Cannot read properties of undefined (reading 'axios')
    at get axios (index.mjs:147:86)
    at Request.request (index.mjs:217:38)
    at Request.get (index.mjs:186:17)
    at Request.fetchById (User.js:18:23)
    at App.vue:10:45
    at runtime-core.esm-bundler.js:1808:40
    at callWithErrorHandling (runtime-core.esm-bundler.js:195:19)
    at callWithAsyncErrorHandling (runtime-core.esm-bundler.js:202:17)
    at hook.__weh.hook.__weh (runtime-core.esm-bundler.js:1788:19)
    at flushPostFlushCbs (runtime-core.esm-bundler.js:379:32)

**Reply by CodeDredd** [reply]
I dont get this error:
https://stackblitz.com/edit/vue3-vite-starter-xuxwzi?file=src%2FApp.vue

**Reply by starredev** [reply]
Sorry @CodeDredd !! Had the wrong dependency, the previous version before your latest while I was thinking i've upgraded it. I'm gonna play with it today! I'm so greatfull for your fix, thank you so much!
